.tekton/serverless-index-pull-request.yaml

```diff
@@ -181,7 +181,7 @@ spec:
       - name: CONTEXT
         value: $(params.path-context)
       - name: HERMETIC
-        value: "false"
+        value: "true"
```

Comment:
This should not be changed

/hold

Reply:
is the hermetic builds is something we want to track for the future?